---
title: Frequently asked questions
---

If you have questions not answered here, please raise a github issue or contact us directly!

### Our goal

As outline in the [BPC white paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02975-0):

> The goal of the BPC is to provide genome assemblies and a community-agreed pangenome representation to replace breed-specific reference assemblies for cattle genomics. The BPC invites partners sharing our vision to participate in the production of these assemblies and the development of a common, community-approved, pangenome reference as a public resource for the research community.

### Why pangenomes?

Using a single reference genome is a broadly powerful approach, standardising a set of genomic coordinates for everyone to use. However, many individuals will contain variation not present in the reference, which can lead to poorly aligned or even unmapped reads. The more diverged the individual, the stronger the ["reference bias"](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03240-8).

Pangenomes aim to incorporate diversity present within a breed/species/genus/etc, such that every sample will have similar alignment quality. By using a pangenome rather than breed/species-specific reference genome, we ensure that the bovine community can still use a standardised set of (pangenomic) coordinates and that research can still be compared across projects.

### Using pangenomes

Using pangenomes is still a rapidly evolving topic. While many useful tools exist for building, visualising, and analysing pangenomes, downstream use of pangenome graphs is still limited. Increasing usibility of pangenomes will be a key priority over the next several years.

### What we want to use the data for

We will produce reference pangenomes and other pangenomic resources using the contributed assemblies. The resulting pangenomes will help mitigate reference bias and better represent global bovine diversity.

### What we won't use the data for

We are not specifically examining individual assemblies or geographic regions in detail. As such, contributing samples/assemblies to the BPC should not impact the "publishing novelty" of any given sample. There is the opportunity for regional working groups to work on complementary/companion papers.

### Contributing to the BPC

We welcome contribution of bovine assemblies (not just cattle!) to the BPC for inclusion into the reference pangenome. Currently we have many assemblies from different
 - sequencing technologies
 - assemblers
 - scaffolding method
 - haplotype-resolution

So reach out if you would like to discuss contributing your assembly.

There is also the possibility to provide a biological sample, which would then be sequenced (and potentially assembled) by the BPC itself. The raw data must become public, but you could retain the "ownership" of the data.

### Minimum contribution to join the BPC

**Nothing!**

In addition to contributing samples/assemblies, we also welcome people purely contributing to the analyses, so reach out if you are interested.

### "Open source" research

Accessibility is a key priority of the BPC, and we are striving to make all of our workflows open-source. Furthermore, we are committed to ensuring all contributed assemblies are publicly accessible at the time of publication to promote open science amongst the community.

### Data access

Since many of the assemblies are not yet public or have restrictions, we currently have controlled access to the data. Members of the BPC can request access to the underlying data, under the explicit condition that any analyses on non-public data must be contributed to the BPC.

### Next data freeze

We are currently estimating the second data freeze to take place in summer 2025, but will depend on the rate we gather new assemblies.

### Outreach

We also want to encourage and support the bovine research community (and beyond) to adopt pangenome references and other emerging techniques.

In 2024 [Paolo Ajmone Marsan](https://docenti.unicatt.it/ppd2/en/docenti/12543/paolo-ajmone-marsan) and [John Williams](https://docenti.unicatt.it/ppd2/en/docenti/81487/john-lewis-williams) organised a [livestock pangenome workshop](https://github.com/ASLeonard/pangenome_summer_school) at the Universit√† Cattolica del Sacro Cuore in Piacenza.

Check back for future opportunities for outreach or workshop activities, or reach out if you would like support/ideas in organising any events.
